spec/services/waste_exemptions_engine/edit_completion_service_spec.rb

```diff
@@ -60,39 +57,37 @@ module WasteExemptionsEngine
         registration.registration_exemptions.first.update(state: "foo")
         edit_registration.exemptions << registration.exemptions.first
```

Comment:
I was wondering why this test didn't catch the original issue with inactive exemptions? I guess it's because we are copying the new state to the `edit_registration` so it's not an accurate replication of what happens. Either way I wonder if we should change it now?

Reply:
This is because the original registration and the copy registration always loaded different exemptions. The registration is using its `complete` trait which initializes it with 10 exemptions. Then the edit was initialized with its `edited` trait, which assigned 5 completely different exemptions to the edit_registration :) (we have made subsequent changes which created this, the exemptions of the `competed` registration, for example, were merged after the edit functionality I believe :D)

Reply:
Cool, I think that's what I thought was going on. I was wondering if we should just get rid of the test now as the line where we copy the exemption to the `edit_registration` no longer reflects what actually happens – it's not an active status so it would not get copied over in the first place. So it seems a bit pointless now to me. What do you think?

Reply:
Good point!! I'll go with the removal of exemptions copy over :) sounds the most reasonable solutions :) We can always re-implement it when / if we decide that also exemptions can be edited :)